src/app/contextBroker/contextBroker.cpp

```diff
@@ -602,15 +605,19 @@ PaArgument paArgs[] =
 
 
 
-#define API_V2                                                                                                   \
-  { "GET",    EPS,       EPS_COMPS_V2,         ENT_COMPS_WORD,      entryPointsTreat                          }, \
-  { "*",      EPS,       EPS_COMPS_V2,         ENT_COMPS_WORD,      badVerbAllFour                            }, \
-  { "GET",    ENT,       ENT_COMPS_V2,         ENT_COMPS_WORD,      getEntities                               }, \
-  { "*",      ENT,       ENT_COMPS_V2,         ENT_COMPS_WORD,      badVerbGetOnly                            }, \
-  { "GET",    IENT,      IENT_COMPS_V2,        IENT_COMPS_WORD,     getEntity                                 }, \
-  { "*",      IENT,      IENT_COMPS_V2,        IENT_COMPS_WORD,     badVerbGetOnly                            }, \
-  { "GET",    IENTATTR,  IENTATTR_COMPS_V2,    IENTATTR_COMPS_WORD, getEntityAttribute                        }, \
-  { "*",      IENTATTR,  IENTATTR_COMPS_V2,    IENTATTR_COMPS_WORD, badVerbGetOnly                            }
+#define API_V2                                                                                     \
+  { "GET",    EPS,       EPS_COMPS_V2,         ENT_COMPS_WORD,       entryPointsTreat           }, \
```

Comment:
(The comment is actually general to the PR... it has been "randomly" done in this file/line to follow our policy of not comenting in the PR itself for the sake of traceability).

Just to be clear: I'm referring to make changes in the .md file explaining how it works V2 in this aspect, no changes in the .cpp or .h files.